Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetGeometryAlgorithms.inl

```diff
@@ -895,44 +866,22 @@ void EdgeSetGeometryAlgorithms<DataTypes>::initPointAdded(PointID index, const c
 
 template<class DataTypes>
 bool EdgeSetGeometryAlgorithms<DataTypes>::computeEdgeSegmentIntersection(EdgeID edgeID,
```

Comment:
Is this method really necessary ? It does exactly the same as the one before, the only difference being that here we pass the reference to the baryCoord object.

Reply:
yes I let it for backward compatibility. But I have added the deprecated macro already in the .h:
SOFA_ATTRIBUTE_DEPRECATED("v23.12", "v24.06", "Use the method computeEdgeSegmentIntersection returning a Coord")